### ISSUE TITLE
Author profile image too wide, hides adding options

### ISSUE THREAD

**Issue by ghost** [Readarr/Readarr#1519]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Searching for an author finds the author. 
When trying to add said author, the "profile image" of the author is so wide that it hides all options for adding. 

I can see "Monitor:" but not the field where I set the monitor value

### Expected Behavior

I expected readarr to see that the image is to wide, and either make it smaller or not display at all
I expected to be able to set the adding options correctly

### Steps To Reproduce

1: Search for author "Michael Dalton"
2: The correct option has a profile image with multiple books in a row
3: Click the author, and the following screen should not show the adding options

![bilde](https://user-images.githubusercontent.com/11895930/152775684-8d80cd20-e522-4d5b-8732-d3fabfcad012.png)

### Environment

```markdown
- OS: Windows 10
- Readarr: 0.1.0.1175
- Docker Install: NO
- Using Reverse Proxy: No
- Browser: Firefox 96.0.3 (64-bit)
```

### What branch are you running?

Develop

### Trace Logs?

2022-02-07 12:04:50.7|Trace|Http|Res: 3634 [GET] /api/v1/rootFolder: 200.OK (14 ms)
2022-02-07 12:04:50.7|Debug|Api|[GET] /api/v1/rootFolder: 200.OK (14 ms)
2022-02-07 12:04:53.7|Trace|Http|Req: 3635 [GET] /api/v1/search?term=Michael%20 (from 127.0.0.1)
2022-02-07 12:04:53.7|Trace|HttpClient|Req: [GET] https://www.goodreads.com/book/auto_complete?format=json&q=michael
2022-02-07 12:04:53.7|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2022-02-07 12:04:54.4|Trace|HttpClient|Res: [GET] https://www.goodreads.com/book/auto_complete?format=json&q=michael: 200.OK (730 ms)
2022-02-07 12:04:54.4|Trace|ConfigService|Using default config value for 'metadatasource' defaultValue:''
2022-02-07 12:04:54.4|Trace|HttpClient|Req: [POST] https://api.bookinfo.club/v1/book/bulk
2022-02-07 12:04:54.4|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2022-02-07 12:04:54.7|Trace|Http|Req: 3636 [GET] /api/v1/search?term=Michael%20Dalton (from 127.0.0.1)
2022-02-07 12:04:54.7|Trace|CachedHttpResponseService|Returning cached response for [GET] https://www.goodreads.com/book/auto_complete?format=json&q=michael%20dalton
2022-02-07 12:04:54.7|Trace|ConfigService|Using default config value for 'metadatasource' defaultValue:''
2022-02-07 12:04:54.7|Trace|HttpClient|Req: [POST] https://api.bookinfo.club/v1/book/bulk
2022-02-07 12:04:54.7|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2022-02-07 12:04:55.8|Trace|HttpClient|Res: [POST] https://api.bookinfo.club/v1/book/bulk: 200.OK (1108 ms)
2022-02-07 12:04:55.8|Trace|Http|Res: 3636 [GET] /api/v1/search?term=Michael%20Dalton: 200.OK (1117 ms)
2022-02-07 12:04:55.8|Debug|Api|[GET] /api/v1/search?term=Michael%20Dalton: 200.OK (1117 ms)
2022-02-07 12:04:57.3|Trace|HttpClient|Res: [POST] https://api.bookinfo.club/v1/book/bulk: 200.OK (2859 ms)
2022-02-07 12:04:57.3|Trace|Http|Res: 3635 [GET] /api/v1/search?term=Michael%20: 200.OK (3609 ms)
2022-02-07 12:04:57.3|Debug|Api|[GET] /api/v1/search?term=Michael%20: 200.OK (3609 ms)
2022-02-07 12:05:00.2|Trace|Http|Req: 3637 [GET] /api/v1/health (from 127.0.0.1)
2022-02-07 12:05:00.2|Trace|Http|Req: 3638 [GET] /api/v1/diskspace (from 127.0.0.1)
2022-02-07 12:05:00.2|Trace|Http|Res: 3637 [GET] /api/v1/health: 200.OK (4 ms)
2022-02-07 12:05:00.2|Trace|Http|Req: 3639 [GET] /api/v1/system/status (from 127.0.0.1)

[AB#2556](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2556)

**Comment by Gykes** [comment]
Can't reproduce this one. Perhaps due to meta being somewhat broken. The image on their Goodreads profile has also changed to align better with other authors that I have seen. 

Chances are some CSS was changed to cap image size at some point.

**Comment by ghost** [comment]
> Can't reproduce this one. Perhaps due to meta being somewhat broken. The image on their Goodreads profile has also changed to align better with other authors that I have seen.
> 
> Chances are some CSS was changed to cap image size at some point.

Considering it otherwise never got a response, and is now a over three year old issue, who knows.

**Comment by mynameisbogdan** [comment]
Currently not an issue anymore, but the previous author image was this wide one. The issue was GR/photo/author/9788458.Michael_Dalton?page=1&photo=1102104

![Image](https://github.com/user-attachments/assets/f050e17b-7c57-4222-bdf7-9fad580699ce)

**Comment by Gykes** [comment]
Okay, perhaps meta is being odd because when I add that author I get no image at all. I can modify the CSS to limit the size of the imported images.

**Comment by mynameisbogdan** [comment]
Not quite, I just did an image override to the image image to show how it looked.

**Comment by Gykes** [comment]
Yep, can reproduce. Easy enough fix.

**Comment by Gykes** [comment]
So there are 2 potential fixes that I can think of. 

1: I just shrink the img. It looks odd as it is a large image and you can't really tell what it is.

2: I make the container flexible so that the banner can go fullsize above all the options and the modal becomes scrollable

Obviously I would center the banner style but if i'm being honest I don't like it too much. Probably just be simpler to just make the image smaller on the modal.


EDIT: I was able to get the image up to a max of around 370px before it started messing with the modal too much. Will probably just go with that

![Small Image](https://github.com/user-attachments/assets/3dff039a-be6b-4e07-b97e-1f47e9bffa45)

![Image](https://github.com/user-attachments/assets/771b89bc-5aaf-4082-9ded-a63f32bb8648)
